refactor(angular): datepicker merge (#825)
sbb-design-systems/sbb-angular: src/angular/migration-guide.md

[Documentation](angular/components/checkbox-panel)

### Datepicker

_Automatic migration available_

> Mainly, the Datepicker remains as it is. As a small change, `SBB_BUSINESS_DATE_ADAPTER` was renamed to `SBB_LEAN_DATE_ADAPTER`.

Reviewer: ```suggestion
The Datepicker has no significant changes. `SBB_BUSINESS_DATE_ADAPTER` was renamed to `SBB_LEAN_DATE_ADAPTER`.
```
